website/content/commands/index.mdx

```diff
@@ -55,26 +55,58 @@ For example, to see help about the up subcommand:
 $ waypoint up -h
 Usage: waypoint up [options]
 
-  Perform the build, deploy, and release steps for the project.
+  Perform the build, deploy, and release steps.
 
 Global Options:
 
+  -app=<string>
+      App to target. Certain commands require a single app target for Waypoint
+      configurations with multiple apps. If you have a single app, then this
+      can be ignored. This is aliased as "-a".
+
+  -plain
+      Plain output: no colors, no animation. The default is false.
+
+  -project=<string>
+      Project to target. This is aliased as "-p".
+
   -workspace=<string>
-      Workspace to operate in. Defaults to 'default'. The default is default.
```

Comment:
I'd vote for us to leave the default value documented here (but not say it twice haha)